packages/app-store/templates/audit-log-implementation/lib/AuditLogManager.ts

```diff
@@ -0,0 +1,28 @@
+import { getGenericAuditLogClient } from "@calcom/features/audit-logs/lib/getGenericAuditLogClient";
```

Comment:
GenericAuditLogManager is just a wrapper around whatever client you give it. This is the default manager which can and should be updated to make it useful. Out of the box it will only console.log whatever event its been given.